### PR TITLE
Created the first draft of the read me file

### PR DESCRIPTION
**Pull request by 95Rawan**
- The file included a small description of the app.
- The motivation behind creating this app.
- A link to the app design on Figma.
- Both the user journey and user stories.
- And a list of the names and GitHub profiles of the team members that are working on the app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,62 @@
...
+
+## [App's UI & UX design on Figma](https://www.figma.com/file/5QhpkfIT570ZyHuMu27U7K/GitGitChat?node-id=0%3A1)
+
+<br/>
+
+## User Journey
+
+- `As a user` of the Github platform that can't chat directly on it.
+
+- `I want to` use GitChat to communicate with other users on Github, wether I worked with them before on the platform or not.
```
Comment (Jood80) [inline README.md]: whether*

File: README.md [hunk 2]
```diff
@@ -1 +1,62 @@
...
+
+<br/>
+
+## User Story
+
+- **As a user**, I expect to see a landing page with the app's logo and name for a few seconds, then be directed to the next page.
+
+- **As a user**, I expect to see a login page that gives me the two options, either to sign in with Github if I have an account, or to click on a link that directs me to the Github sign up page.
+
+- **As a user**, I expect to be directed to a page with multiple screens after I sign in. The default screen I'll land on is the chatting screen which will have a `navigation bar` that displays the `app's logo` on the left and the `app's name` on the right, in the middle section of the screen there will be a `sad lonely person image` and a `start a new chat button` if the person doesn't have previous chats, at the bottom section of the screen there will be a footer that will include -along with the `chat option`-, a `contacts option` and a `profile option`.
```
Comment (Jood80) [inline README.md]: - "multiple screens", shouldn't that be tabs instead?
- we have changed it from "lonely person" into a " lonely octocat 
- the position of both the icon and the App name in the header has been changed to be beside each other to the left
- the navigation bar is on screen footer

**Review by Jood80 — APPROVED** [review]

**Review by Rahaf-96 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (95Rawan) [reply README.md]: Done

File: README.md (hunk 1 above)
Comment (95Rawan) [reply README.md]: Done

**Review by bayan-404 — APPROVED** [review]

**Pull request merged**